Kconfig.zephyr

```diff
@@ -50,6 +50,7 @@ source "drivers/Kconfig"
 source "lib/Kconfig"
 source "subsys/Kconfig"
 
+source "cmake/toolchain/Kconfig"
```

Comment:
please remove, we are including `$(TOOLCHAIN_KCONFIG_DIR)/Kconfig` right below, which allows the toolchain itself to adjust defined Kconfig.

Content of file is commented elsewhere.

Reply:
Moved.